public/app/features/transformers/docs/content.ts

```diff
@@ -0,0 +1,1024 @@
+/*
+  NOTE: This file is used to generate the transformation docs markdown content. If you change/update the content here,
+  please then rebuild the markdown by doing the following:
+
+  $ cd /docs (from the root of the repository)
+  $ make sources/panels-visualizations/query-transform-data/transform-data/index.md
+  $ make docs
+
+  Browse to http://localhost:3003/docs/grafana/latest/panels-visualizations/query-transform-data/transform-data/
+
+  Refer to ./docs/README.md for more information about building docs. 
+*/
+
+interface Link {
+  title: string;
+  url: string;
+}
+export interface TransformationInfo {
+  name: string;
+  helperDocs: string;
+  links?: Link[];
+}
+
+export const transformationDocsContent: Record<string, TransformationInfo> = {
+  calculateField: {
+    name: 'Add field from calculation',
+    helperDocs: `
+  Use this transformation to add a new field calculated from two other fields. Each transformation allows you to add one new field.
+
+  - **Mode -** Select a mode:
+    - **Reduce row -** Apply selected calculation on each row of selected fields independently.
+    - **Binary option -** Apply basic math operation(sum, multiply, etc) on values in a single row from two selected fields.
+    - **Index -** Will insert a field with the row index.
+  - **Field name -** Select the names of fields you want to use in the calculation for the new field.
+  - **Calculation -** If you select **Reduce row** mode, then the **Calculation** field appears. Click in the field to see a list of calculation choices you can use to create the new field. For information about available calculations, refer to [Calculation types][].
+  - **Operation -** If you select **Binary option** mode, then the **Operation** fields appear. These fields allow you to do basic math operations on values in a single row from two selected fields. You can also use numerical values for binary operations.
+  - **Alias -** (Optional) Enter the name of your new field. If you leave this blank, then the field will be named to match the calculation.
+  - **Replace all fields -** (Optional) Select this option if you want to hide all other fields and display only your calculated field in the visualization.
+  `,
+    links: [
+      {
+        title: 'Calculation types',
+        url: 'https://grafana.com/docs/grafana/latest/panels-visualizations/calculation-types/',
+      },
+    ],
+  },
+  concatenate: {
+    name: 'Concatenate fields',
+    helperDocs: `
+  Use this transformation to combine all fields from all frames into one result. Consider the following:
```

Comment:
What gets pulled into the UI in this case?

Reply:
Yep, all the content under the `helperDocs` key is pulled in and displayed. The UI work is in another PR, but I attached is what this will look like ->
<img width="767" alt="Screenshot 2023-10-18 at 1 11 37 PM (2)" src="https://github.com/grafana/grafana/assets/46619047/a69cd27a-2184-42a2-b135-51144cc9d6cd">

Reply:
Okay this is helpful to see. I couldn't imagine it so all the content felt way too long. Thanks for posting!